orders/handlers/create.js

```diff
@@ -0,0 +1,10 @@
+const OrdersController = require('../controller');
+const Model = require('../models/OrderModel');
+const collectionHandler = require('../../util/collectionHandler');
+
+module.exports = [
+  collectionHandler(Model),
```

Comment:
Create shoudln't need the collectionHandler : we will never pass down the page/pagesize to this endpoint